content/300-guides/150-testing/100-unit-testing.mdx

```diff
@@ -84,7 +84,7 @@ The following steps guide you through mocking the Prisma Client using a singleto
 
    ```ts file=singleton.ts
    import { PrismaClient } from '@prisma/client'
-   import { mockDeep, mockReset, MockProxy } from 'jest-mock-extended'
+   import { mockDeep, mockReset, MockDeepProxy } from 'jest-mock-extended'
```

Comment:
This interface doesnt seem to exist on the `jest-mock-extended` package...

Reply:
Ah, sorry, this is not the import I was using. I corrected it!